src/sass/layout/modals/_franchise.scss

```diff
@@ -0,0 +1,242 @@
+.backdrop {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  opacity: 1;
+  transition: opacity 250ms cubic-bezier(0.4, 0, 0.2, 1),
+    visibility 250ms cubic-bezier(0.4, 0, 0.2, 1);
+}
+
+.is-hidden {
+  opacity: 0;
+  pointer-events: none;
+  visibility: hidden;
+}
+
+.franchise__modal {
+  position: absolute;
+  top: 55%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(1);
+  padding: 28px;
+  border-radius: 10px;
+  border: 2px solid #f8d24e;
+  background-color: getColor('white-contacts');
+  transition: transform 250ms cubic-bezier(0.4, 0, 0.2, 1);
+
+  .is-hidden & {
+    transform: translate(-50%, -50%) scale(0);
+  }
+
+  @media screen and (max-width: 767px) {
+    width: 280px;
+    height: 674px;
+  }
+
+  @include tablet {
+    width: 568px;
+    height: 668px;
+    padding: 32px 60px;
+  }
+}
+
+.franchise__button {
+  position: absolute;
+  left: 90%;
```

Comment:
Seems like the icon isn't located properly.
can we try to add the next code?
top: 12px;
left: 12px;

Reply:
left: 90%;
  top: 12px;